zebra-network/src/address_book.rs

```diff
@@ -463,6 +556,23 @@ impl AddressBook {
         self.by_addr.descending_values().cloned()
     }
 
+    /// Is this IP ready for a new outbound connection attempt?
+    /// Checks if the most recently updated outbound address with this IP `was_recently_updated()`.
```

Comment:
If we just look at last response, these docs and code need to be updated.


```suggestion
    /// Checks if the most recently updated outbound address with this IP `has_connection_recently_responded()`.
```

Reply:
Applied something similar in https://github.com/ZcashFoundation/zebra/pull/7029/commits/788bce3b9ea89d3d5b3d535d09720b9b04c169a0.